RRS/customer/urls.py

```diff
@@ -10,7 +10,7 @@
         name  = 'index'
     ),
     url(
-        regex = r'restaurant/(\d+)$',
+        regex = r'restaurant/(\d+)/$',
```

Comment:
I'm checking the webpages open in my tabs now, but they only seem to use "/" for directories and without "/" for most urls